espnet2/asr/encoder/conformer_encoder.py

```diff
@@ -93,14 +93,17 @@ def __init__(
         positionwise_layer_type: str = "linear",
         positionwise_conv_kernel_size: int = 3,
         macaron_style: bool = False,
-        rel_pos_type: str = "legacy",
```

Comment:
@pengchengguo, please check it.
I think it will break compatibility.

Reply:
Rel pos type can be whatever the ESP default is, I don't need to amend it since I am not using it- will fix